src/content/app/tools/blast/components/blast-input-sequences/BlastInputSequencesHeader.tsx

```diff
@@ -46,10 +49,21 @@ const BlastInputSequencesHeader = (props: Props) => {
   const { sequences, sequenceType, updateSequenceType, clearAllSequences } =
     useBlastForm();
 
-  const isEmptyInputAppended = useSelector(getEmptyInputVisibility);
-  const isUserTypingInEmptyInput = useSelector(getUncommittedSequencePresence);
+  const isEmptyInputAppended = useAppSelector(getEmptyInputVisibility);
+  const isUserTypingInEmptyInput = useAppSelector(
+    getUncommittedSequencePresence
+  );
+
+  const dispatch = useAppDispatch();
 
-  const dispatch = useDispatch();
+  const sequenceValidityContext = useContext(BlastFormContext);
+  let shouldShowAlert = false;
+
+  if (sequenceValidityContext) {
+    shouldShowAlert = Object.values(
+      sequenceValidityContext.sequenceValidityStatus
+    ).includes(false);
+  }
```

Comment:
Another, more concise way to write this would be:

```  const shouldShowAlert = sequenceValidityContext && Object.values(
    sequenceValidityContext.sequenceValidityStatus
  ).includes(false);```